embassy-rp/src/usb.rs

```diff
@@ -567,14 +580,17 @@ impl<'d, T: Instance> driver::EndpointOut for Endpoint<'d, T, Out> {
         let index = self.info.addr.index();
         let val = poll_fn(|cx| {
             EP_OUT_WAKERS[index].register(cx.waker());
+            let ctrl_val = T::dpram().ep_out_control(index - 1).read();
```

Comment:
I could not find any documentation that says why this has to be `- 1`, but it is what the `wait_enabled` functions do so I figure it must be correct.